customers.go

```diff
@@ -128,6 +128,14 @@ type MergeCustomersParams struct {
 	Into CustID `json:"into"`
 }
 
+// UnmergeCustomersParams - identify target for unmerging.
+type UnmergeCustomersParams struct {
+	CustomerUUID      string   `json:"customer_uuid"`
+	ExternalID        string   `json:"external_id"`
+	DataSourceUUID    string   `json:"data_source_uuid"`
+	MoveToNewCustomer []string `json:"move_to_new_customer,omitempty"`
```

Comment:
[UX] Add a constant with all the options for convenience?

Reply:
I'm hesitant to do this everywhere, because if we add constants like this to all of the SDKs then we'll have to update all of them if/when a new object gets added to the list. A better solution will probably be to update the api endpoint to accept a value like "all" that triggers this behavior, regardless of which SDK is being used. I'll look into that separately.

Reply:
@jessicahearn that's a good point and a good solution :+1: